structures/ansible_structures.plantuml

```diff
@@ -0,0 +1,27 @@
+@startuml
+Landscape *- Type
+Type *- Function
+Function *- Component
+class Landscape << (L,orchid) >> {
+	{field} Workflow
+	{method} Playbook of playbooks
+}
+class Type << (T,orange) >> {
+	Playbook
+	{method} _
+}
+class Function << (F,yellow) >> {
+	Role
+	{method} _
+}
+class Component {
+	{field} Task file
+	{method} Role
+}
+hide empty members
+scale 750 width
+skinparam classBackgroundColor Wheat/CornflowerBlue
```

Comment:
I do like the colours, but I am afraid they aren't great in terms of accessibility.

Reply:
And I liked so much that I found a way to have PlantUML graphs look good.

Reply:
I used lighter colors...